app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationController.scala

```diff
@@ -93,59 +93,47 @@ class BusinessIdentificationController @Inject()(
         .bindFromRequest()
         .fold(
           formWithErrors => {
-            if (formWithErrors.errors.exists(_.message == "error.business-type-value.invalid")) {
-              Logger.warn("Select business-type form submitted with invalid identifier")
-              throw new BadRequestException("submitted form value did not contain valid businessType identifier")
-            }
             Future successful Ok(html.business_type(formWithErrors))
           },
-          validatedBusinessType =>
+          validatedBusinessType => {
             Future successful Redirect(
               routes.BusinessIdentificationController.showBusinessDetailsForm(validatedBusinessType.businessType))
+          }
         )
     }
   }
 
-  def showBusinessDetailsForm(businessType: Option[String]): Action[AnyContent] = Action.async { implicit request =>
-    withSubscribingAgent { implicit agent =>
-      withMaybeContinueUrlCached {
-        businessType match {
-          case Some(businessTypeIdentifier) if validBusinessTypes.contains(businessTypeIdentifier) => {
-            mark("Count-Subscription-BusinessDetails-Start")
-            Future successful Ok(html.business_details(knownFactsForm(businessTypeIdentifier), businessTypeIdentifier))
-          }
-          case _ => {
-            Logger.warn("businessTypeIdentifier was missing, redirect and obtain from showCheckBusinessType page")
-            Future successful Redirect(routes.BusinessIdentificationController.showBusinessTypeForm())
-          }
+  def showBusinessDetailsForm(
+    businessType: IdentifyBusinessType = IdentifyBusinessType.SoleTrader): Action[AnyContent] = Action.async {
```

Comment:
This deafults to SoleTrader, but I believe the previous behaviour was deliberately redirecting to the showBusinessTypeForm page in order to support those UI journeys that are still linking directly to the old URL (from a time before we created the show business type page).